packages/edit-site/src/store/controls.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * WordPress dependencies
+ */
+import { createRegistryControl } from '@wordpress/data';
+
+/**
+ * Calls a selector using the current state.
+ *
+ * @param {string} storeName    Store name.
+ * @param {string} selectorName Selector name.
+ * @param  {Array} args         Selector arguments.
+ *
+ * @return {Object} control descriptor.
+ */
+export function select( storeName, selectorName, ...args ) {
```

Comment:
Where does this export get used?  
I see the default export in this file gets used in `registerStore` in the index, but what is this function doing?

Reply:
I copied over the scaffolding from `edit-post` store to speed things up so likely an unintended leftover. Will recheck and remove it if it's not necessary. I also have to look into failing CI tests.

Reply:
Removed in 1ce8f0db2d7503f5c7fe967ad9b53e00b01bf379.